includes/definitions/edgeos.yaml

```diff
@@ -11,3 +11,8 @@ discovery:
     - sysDescr_regex:
         - '/^EdgeOS/'
         - '/^EdgeRouter/'
+bad_if_regexp:
+    - '/^npi[0-9]/'
```

Comment:
The others make sense but what type of port is this?

Reply:
from ubnt forums: 

Those are additional devices created by the Ethernet driver (only on the ERPro-8 and ER-8) and are not used by the system, so they can be ignored.